Script/AtomicNET/AtomicNETService/CSComponentInspector.cs

```diff
@@ -441,21 +441,28 @@ internal static bool CrackStringInAttributeValue(out string value, ref BlobReade
 
                             var fieldDef = metaReader.GetFieldDefinition(fieldHandle);
 
-                            var fieldName = metaReader.GetString(fieldDef.Name);
-
-                            if (opCode.ToString() == "stfld")
+                            // No way to predetermine if fieldDef.Name is valid
+                            try
                             {
+                                var fieldName = metaReader.GetString(fieldDef.Name);
 
-                                InspectorField inspectorField;
-
-                                if (_inspectorComponent.Fields.TryGetValue(fieldName, out inspectorField))
+                                if (opCode.ToString() == "stfld")
                                 {
-                                    inspectorField.DefaultValue = String.Join(" ", loadedValues.ToArray());
-                                }
 
-                            }
+                                    InspectorField inspectorField;
 
-                            loadedValues.Clear();
+                                    if (_inspectorComponent.Fields.TryGetValue(fieldName, out inspectorField))
+                                    {
+                                        inspectorField.DefaultValue = String.Join(" ", loadedValues.ToArray());
+                                    }
+
+                                }
+                            }
+                            catch (Exception) { break; }
+                            finally
```

Comment:
I don't know. The only semi-useful info would be the class name, which could be passed down from the calling method, but even that isn't really useful since it happens on classes that have no constructor or inline assignments. If there was something we could actually report saying "fix this to make the message go away", then fine, but that doesn't seem to be the case.